src/main/scala/com.github.mgifos.workouts/Main.scala

```diff
@@ -26,6 +26,7 @@ case class Config(
   system: MeasurementSystems.MeasurementSystem = MeasurementSystems.metric,
   csv: String = "",
   delete: Boolean = false,
+  `import`: Boolean = true,
```

Comment:
No need for introducing `import` as config parameter again, since it's already available as `config.import`